src/librustdoc/clean/mod.rs

```diff
@@ -2263,7 +2263,7 @@ impl Clean<Vec<Item>> for doctree::Import<'_> {
             name: None,
             attrs: self.attrs.clean(cx),
             source: self.span.clean(cx),
-            def_id: DefId::local(CRATE_DEF_INDEX),
+            def_id: cx.tcx.hir().local_def_id(self.id).to_def_id(),
```

Comment:
I'm not sure how this will affect the rest of rustdoc. @GuillaumeGomez do you know how the DefId is used for import items?

Reply:
I guess if it doesn't break the tests it's probably fine?

Reply:
I'd think that if no test is broken, it's fine too.